Source/OCMock/NSInvocation+OCMAdditions.m

```diff
@@ -106,18 +114,9 @@ - (void)retainObjectArgumentsExcludingObject:(id)objectToExclude
     {
         id returnValue;
         [self getReturnValue:&returnValue];
-        if((returnValue != nil) && (returnValue != objectToExclude))
+        if (returnValue != nil)
         {
-            if(OCMIsBlockType(returnType))
-            {
-                id blockReturnValue = [returnValue copy];
-                [retainedArguments addObject:blockReturnValue];
-                [blockReturnValue release];
-            }
-            else
-            {
-                [retainedArguments addObject:returnValue];
-            }
+            [NSException raise:NSInternalInconsistencyException format:@"Return value should never be set"];
```

Comment:
See comment above. Is this really related to the feature this PR wants to add?

Reply:
Removed as well.